.env-sample

```diff
@@ -0,0 +1 @@
+UPGRADER_VERSION=1.3.0
```

Comment:
Might be better to rename `.env-sample` to `.env.example` as thats what we use in other repos.

Reply:
done.